config/environments/production.rb

```diff
@@ -80,4 +80,17 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # mandrill config stuff, depends on config/initializers/constants.rb
+  config.action_mailer.smtp_settings = {
```

Comment:
Can this be moved into an initializer? If we're using environment variables, we should only have to configure the SMTP settings once at application startup.